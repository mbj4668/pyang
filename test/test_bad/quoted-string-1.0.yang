module quoted-string-1.0 {
  namespace urn:example:qs;
  prefix qs;

  description
    "Newline is ok.\nAnd tab\t, and a slash\\, and a
     double quote \".";

  container x {
    description
      "But nothing else! \'."; // this is a warning
  }
}

