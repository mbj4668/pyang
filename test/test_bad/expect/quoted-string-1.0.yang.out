quoted-string-1.0.yang:11: warning: ILLEGAL_ESCAPE_WARN
